examples/gui/main.rs

```diff
@@ -0,0 +1,236 @@
+#![allow(irrefutable_let_patterns)]
+
+use blade_graphics as gpu;
+use winit::platform::x11::WindowAttributesExtX11;
+
```

Comment:
do we need a separate example for this? we already have `particle` using egui, `scene` using it, and `move` as well.

Reply:
Nop, will remove, didnät intend to include it here, was just the one I used to debug